contracts/staking/contracts/src/libs/LibFixedMathRichErrors.sol

```diff
@@ -0,0 +1,97 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/LibRichErrors.sol";
+
+
+library LibFixedMathRichErrors {
+
+    enum ValueErrorCodes {
+        TOO_SMALL,
+        TOO_LARGE
+    }
+
+    enum BinOpErrorCodes {
+        ADDITION_OVERFLOW,
+        SUBTRACTION_OVERFLOW,
+        MULTIPLICATION_OVERFLOW,
+        DIVISION_BY_ZERO
+    }
+
+    // bytes4(keccak256("FixedMathSignedValueError(uint8,int256)"))
+    bytes4 internal constant SIGNED_VALUE_ERROR_SELECTOR =
+        0x0edd9c46;
+
+    // bytes4(keccak256("FixedMathUnsignedValueError(uint8,uint256)"))
+    bytes4 internal constant UNSIGNED_VALUE_ERROR_SELECTOR =
+        0x38e93856;
+
+    // bytes4(keccak256("FixedMathBinOpError(uint8,int256,int256)"))
+    bytes4 internal constant BIN_OP_ERROR_SELECTOR =
+        0x2bd3386e;
+
+    // solhint-disable func-name-mixedcase
+    function FixedMathSignedValueError(
```

Comment:
optional nit: remove the `FixedMath` prefix from these functions (`LibFixedMathRichErrors.FixedMath...Error` is a bit redundant)